storage.go

```diff
@@ -22,6 +23,25 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 	return o
 }
 
+func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
+	rp := s.getAbsPath(path)
+
+	options := make([]oss.Option, 0)
+	options = append(options, oss.ContentLength(0))
+
+	offset, err := s.bucket.AppendObject(rp, strings.NewReader(""), 0, options...)
```

Comment:
Can we use `nil` instead of `strings.NewReader("")` here?

Reply:
Maybe the GetReaderLen() in https://github.com/aliyun/aliyun-oss-go-sdk/blob/master/oss/utils.go#L438 will return error if we use nil?

Reply:
Let's give it a try. Our integration tests will cover it.